Helper/Log.php

```diff
@@ -91,14 +91,15 @@ public function __construct(
      * @param string $message
      * @param string $type
      */
-    public function log(string $module, string $action, string $message, string $type)
+    public function log(string $module, string $action, string $message, string $type, string $context)
```

Comment:
@lewisvoncken I think this should have a default value. Otherwise its not backwards compatible